gulpfile.js

```diff
@@ -14,8 +17,14 @@ gulp.task('rel', 'Reload task', function () {
     .pipe(connect.reload());
 });
 
+gulp.task('sass', function () {
+  return gulp.src('./**/*.scss')
+    .pipe(sass().on('error', sass.logError))
+    .pipe(gulp.dest('./'));
+});
+
 gulp.task('watch', 'Watch task', function () {
-  gulp.watch(['./**'], ['rel']);
+  gulp.watch(['./**/*.html', './**/*.css', './**/*.scss'], ['rel', 'sass']);
```

Comment:
I would separate this into two lines, with the html and the css triggering the 'rel' task and the scss triggering the 'sass' task. Right now it gets triggered a couple times for sass changes, and you don't really need to run sass if it's triggered by an html change.